src/oc_erchef/apps/chef_objects/src/chef_object_base.erl

```diff
@@ -148,8 +148,13 @@ make_org_prefix_id(OrgId, Name) ->
     %% assume couchdb guid where trailing part has uniqueness
     <<_:20/binary, OrgSuffix:12/binary>> = OrgId,
     Bin = iolist_to_binary([OrgId, Name, crypto:strong_rand_bytes(6)]),
-    <<ObjectPart:80, _/binary>> = crypto:hash(md5, Bin),
+    <<ObjectPart:80, _/binary>> = erlang:md5(Bin),
     iolist_to_binary(io_lib:format("~s~20.16.0b", [OrgSuffix, ObjectPart])).
+    %% Object part is expected to be an integer
+%    ObjectPart = crypto:strong_rand_bytes(10),
```

Comment:
Am I reading this correctly, was this previously all dead code? I'm confused about how this was compiling.

Reply:
This was an attempt to get rid of the erlang:md5 call completely. But I stuck to keeping it there for now. I will remove the newly added unused code.